Matlab.tmbundle/Syntaxes/MATLAB.tmLanguage

```diff
@@ -1454,7 +1454,7 @@
 							<key>begin</key>
 							<string>\G</string>
 							<key>end</key>
-							<string>(?&lt;=[\)\n])</string>
+							<string>(?&lt;=\))|(?&gt;(?&lt;!\.{3}.*)\n)</string>
```

Comment:
Ending the function declaration on `)` or `\n` will not allow for line continuations in the declaration.